docs/source/modules/event.rst

```diff
@@ -0,0 +1,51 @@
+.. _event-module:
+
+Event Module
+============
+
+.. contents::
+    :depth: 2
+
+Description
+-----------
+
+The Event module allows to create listener and assign action to event. The module is using :code:`MotechEvent` to provide
+event with specific data that will be sent by Motech Scheduler when a scheduled event is fired.
+The module is using ActiveMQ to support the event execution and callback. It is possible to register your own callback
+function using :code:`EventCallbackService`.
+
+MotechEvent
+-----------
+
+The Event module use :code:`MotechEvent` to storage information about fired event. Instance of this class is sending by Motech Scheduler
+module to a specific handler (see :code:`TaskTriggerHandler` in Tasks module). Main fields of :code:`MotechEvent`:
```

Comment:
This whole sentence is blatantly incorrect. It doesn't need to be scheduler, the handler does not need to be the TaskTriggerHandler. The Event module is not connected directly to any other MOTECH module. Every single MOTECH module can use MOTECH events to both fire and receive events (so be a publisher and subscriber). Saying that Scheduler fires events or Tasks receive events can suggest that the Event module is limited to those modules.